.github/ISSUE_TEMPLATE

```diff
@@ -0,0 +1,26 @@
+### Description of Issue/Question
```

Comment:
Before this section I would add something like:

```
### Have you performed the steps presented in https://github.com/napalm-automation/napalm/pull/267/files?
[ ] Yes
[ ] No
```

With a link to the new FAQ section in the README.

Yes/No to be filled in with an ```x```.